src/coll/options.rs

```diff
@@ -1,25 +1,39 @@
 //! Options for collection-level operations.
 use bson::{self, Bson};
-use cursor;
 use common::{ReadPreference, WriteConcern};
 use Error::ArgumentError;
 use Result;
 
 /// Describes the type of cursor to return on collection queries.
-#[derive(Clone, PartialEq, Eq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
A lot of these types didn't have `Debug` implemented for them, which made debugging some things a pain (go figure).